### PR TITLE
Allow to use custom track

### PR DESCRIPTION
**Pull request by eeeli24**

### CODE REVIEW

File: DefaultTrack.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                {twoMarkers && (
+                    <View style={[styles.track, this.props.trackStyle, trackThreeStyle, {width: trackThreeLength}]} />
+                )}
+                {children}
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
```
Comment (basvasilich) [inline DefaultTrack.js]: separate file

File: DefaultTrack.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    render() {
+        const {
+            positionOne,
+            positionTwo,
+            sliderLength,
+            twoMarkers,
+            selectedStyle,
+            unselectedStyle,
+            children
+        } = this.props;
```
Comment (sunr1ze) [inline DefaultTrack.js:31]: You use `this.props.trackStyle` 3 times. Why dont you make const for it?

File: MultiSlider.js [hunk 3]
```diff
@@ -294,29 +303,24 @@ export default class MultiSlider extends React.Component {
...
-        const trackThreeLength = twoMarkers ? sliderLength - positionTwo : 0;
-        const trackThreeStyle = unselectedStyle;
-        const trackTwoLength = sliderLength - trackOneLength - trackThreeLength;
-        const trackTwoStyle = twoMarkers ? selectedStyle || styles.selectedTrack : unselectedStyle;
         const Marker = this.props.customMarker;
-
         const MarkerLeft = this.props.customMarkerLeft;
         const MarkerRight = this.props.customMarkerRight;
+
+        const Track = this.props.customTrack;
```
Comment (basvasilich) [inline MultiSlider.js]: you have `const {...} = this.props;` in 305 do not repeat `this.props`

**Review by sunr1ze — APPROVED** [review]

File: MultiSlider.js [hunk 4]
```diff
@@ -328,80 +332,81 @@ export default class MultiSlider extends React.Component {
...
+                                {isMarkersSeparated === false ? (
+                                    <Marker
+                                        pressed={this.state.twoPressed}
+                                        markerStyle={this.props.markerStyle}
+                                        pressedMarkerStyle={this.props.pressedMarkerStyle}
+                                        currentValue={this.state.valueTwo}
+                                        enabled={this.props.enabledTwo}
+                                        valuePrefix={this.props.valuePrefix}
+                                        valueSuffix={this.props.valueSuffix}
+                                    />
```
Comment (basvasilich) [inline MultiSlider.js:423]: the same for `this.props`

**Review by basvasilich — APPROVED** [review]

**Pull request merged**